openforcefield/data/test_forcefields/GBSA_OBC1-1.0.offxml

```diff
@@ -0,0 +1,15 @@
+<SMIRNOFF version="0.3" aromaticity_model="OEAroModel_MDL">
```

Comment:
Can we add some provenance information for each of these experimental `.offxml` files? Where does this come from?

Reply:
Added provenance info in 93f9c93